api/inner_inerfaces.puml

```diff
@@ -0,0 +1,11 @@
+@startuml
+
+interface IMachines {
+
+getAvailableMachines() : List<Machine>
+activateComputationTask(Computation computation, Machine machine) : bool
```

Comment:
Czemu bool zwracany? Co to oznacza?

Reply:
Computation i wybrana Machine przekazywana do Machine do aktywacji? Tak miało to wyglądać? Jest to zdefiniowane w swaggerze? 

POST
​/machine​/computation
run computation on machine

Reply:
> Computation i wybrana Machine przekazywana do Machine do aktywacji? Tak miało to wyglądać?

Właśnie pytam się, czy was recenzentów, czy to rozwiązanie jest dla was wystarczająco wygodne, czy proponujecie jakąś zmianę?

> Jest to zdefiniowane w swaggerze?

Nie, bo ten interfejs nie zakłada komunikacji po HTTP, tylko wywołanie metody wewnątrz jednego modułu, jakim jest __MachineManager__. Zmiany w Swaggerze byłyby zmianami w pliku `api.yml`.